tests/manage/monitoring/prometheus/test_deployment_status.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import tier4
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@pytest.mark.polarion_id("OCS-1052")
+def test_ceph_manager_stopped(workload_stop_ceph_mgr):
+    """
+    Test that there is appropriate alert when ceph manager
+    is unavailable and that this alert is cleared when the manager
+    is back online.
+    """
+    prometheus = PrometheusAPI()
+
+    alerts = workload_stop_ceph_mgr['prometheus_alerts']
+    target_label = 'CephMgrIsAbsent'
+    target_alerts = [
```

Comment:
use .get('') instead of accessing the dict items

Reply:
done